ipatests/test_integration/test_legacy_clients.py

```diff
@@ -326,6 +328,91 @@ def test_login_disabled_subdomain_ad_user(self):
 
         assert result.returncode != 0
 
+    def test_getent_treedomain_ad_user(self):
+        if not self.ad_treedomain:
+            raise nose.SkipTest('AD tree root domain is not available.')
+
+        self.clear_sssd_caches()
+        testuser = 'treetestuser@{0}'.format(self.ad_treedomain)
+        result = self.legacy_client.run_command(['getent', 'passwd', testuser])
+
+        testuser_regex = "treetestuser@%s:\*:%s:%s:"\
```

Comment:
Backslashes in multiline strings are discouraged by PEP-8, please enclose the whole string literal in parentheses and leave out backslashes.

Also please try to use new-style string formatting in new code.